public_html/assets/css/global/style.css

```diff
@@ -283,40 +295,47 @@ html[dir="rtl"] #tabzilla {
 }
 
 .header {
-  background: transparent;
-  margin-top: -50px;
-  padding-top: 0;
+  align-items: center;
+  display: flex;
+  height: 382px;
+  justify-content: center;
+  left: 0;
+  position: absolute;
+  right: 0;
   text-align: center;
+  top: 0;
 }
 
-.header img {
-  -o-animation: fadein 3s;
-  -webkit-animation: fadein 3s;
-  animation: fadein 3s;
-  display: block;
-  font-size: 8rem;
-  margin: 190px auto 0;
+.site-title {
+  -o-animation: fadein 2s;
+  -webkit-animation: fadein 2s;
+  animation: fadein 2s;
+  color: #dcf4fe;
+  font-size: 2.4rem;
+  font-size: calc(2.6rem + 4vw);
+  font-weight: 300;
+  letter-spacing: -.03ch;
+  line-height: 1;
+  margin: 0 auto;
   max-width: 90%;
 }
 
-.header img.headline-mobile {
-  display: none;
+@supports (-webkit-text-fill-color: transparent) {
```

Comment:
I hadn't really seen any feature queries in the wild yet, this is neat

Reply:
I didn't know about them until I started searching online for examples of this property. I didn't know `@supports` had shipped yet! and it's just safely skipped in older browsers 👍